tests/postgres_tests/test_serialfield.py

```diff
@@ -0,0 +1,63 @@
+from django.contrib.postgres.fields.serial import BigSerialField, SerialField
+from django.db import models
+from django.db.backends.postgresql.base import \
+    DatabaseWrapper as PostgresDatabaseWrapper
+from django.test import SimpleTestCase, skipUnlessDBFeature
+from django.test.utils import isolate_apps
+
+from . import PostgreSQLTestCase
+from .models import SerialModel
+
+
+@isolate_apps('invalid_models_tests')
+class SerialFieldModelTests(SimpleTestCase):
+    def test_not_unique(self):
```

Comment:
The method name `test_not_unique` is a bit odd given the field is `unique=True`?